lib/Views/CreatePost.dart

```diff
@@ -3,7 +3,6 @@ import 'package:flutter/material.dart';
 import 'package:flutterassignment/components/pick_image_button.dart';
 import 'package:flutterassignment/components/post_textfield.dart';
 import '../components/post_button.dart';
-import 'package:image_picker/image_picker.dart';
```

Comment:
Need to find a way to integrate image picker back in that isn't just downgrading to Ladybug